nim/src/views/general.nim

```diff
@@ -0,0 +1,22 @@
+#? stdtmpl(subsChar = '$', metaChar = '#')
```

Comment:
```suggestion
#? stdtmpl
```

Reply:
This is what was done in Nim in Action, so I assumed it was standard